projects/components/src/filtering/filter/parser/parsed-filter.ts

```diff
@@ -77,7 +77,7 @@ export const tryParseStringForAttribute = (
     }
     const potentialSubpath = stringAfterAttributeName.slice(MAP_LHS_DELIMITER.length);
     // Subpaths support alphanumeric, -, - and . characters
-    const firstNonSubpathCharacterIndex = potentialSubpath.search(/[^\w\-\.]/);
+    const firstNonSubpathCharacterIndex = potentialSubpath.search(/[^\w\-\.\:]/);
```

Comment:
please update comment (also it says `-` twice - guessing one should be `_` as that is also valid, but it's missing from regex)

Reply:
done